src/Schemes/AffineSchemes/Objects/Properties.jl

```diff
@@ -0,0 +1,487 @@
+export is_empty, issubset, is_open_embedding, is_closed_embedding
+
+
+
+####################################################################################
+# (1) Check if a scheme is empty
+####################################################################################
+
+@Markdown.doc """
+    isempty(X::AbsSpec)
+
+This method returns `true` if the affine scheme ``X`` is empty.
+Otherwise, `false` is returned.
+
+# Examples
+```jldoctest
+julia> X = affine_space(QQ,3)
+Spec of Multivariate Polynomial Ring in x1, x2, x3 over Rational Field
+
+julia> isempty(X)
+false
+
+julia> isempty(EmptyScheme(QQ))
+true
+```
+"""
+Base.isempty(X::AbsSpec) = iszero(one(OO(X)))
+is_empty(X::EmptyScheme) = true
+
+
+
+####################################################################################
+# (2) IsSubset for all other schemes
+####################################################################################
+
+
+# (2.0) For empty schemes and whenever issubset cannot be implemented
+# (2.0) For empty schemes and whenever issubset cannot be implemented
```

Comment:
duplicate line?

Reply:
Yes - thank you. (Seems I considered this a good idea a couple of days ago. But I agree this is not that good. Removed this and three other such lines just now.)